spec/models/saved_recipie_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe SavedRecipie do
+  describe 'validations' do
+    describe 'recipie is sucsessfully saved' do
+      let(:user) { User.create!(email: 'random@test4.com', password: 'password123', password_confirmation: 'password123') }
+
+      it 'recipie is saved sucsessfully' do
+        saved_recipie = described_class.create(user_id: user.id, favorited: true, api_recipie_id: 1)
+        ## The recipie is saved succsessfuly
+        expect(saved_recipie).to be_valid
+        ## The recipie is saved to the user
+        expect(saved_recipie.user_id).to eq(user.id)
+      end
```

Comment:
I like the changes you made to this test! I think it is much cleaner and clearer.